source/MaterialXTest/MaterialXRuntime/Runtime.cpp

```diff
@@ -2222,8 +2222,33 @@ TEST_CASE("Runtime: Commands", "[runtime]")
     mx::RtCommand::undo(result);
     REQUIRE(result);
 
-    REQUIRE(makeConnectionCount == 6);
-    REQUIRE(breakConnectionCount == 2);
+    REQUIRE(connectionChangeCount == 8);
+
+
+    //
+    // Test adding and removing relationship targets
+    //
+
+    mx::RtCommand::createPrim(stage, mx::RtGeneric::typeName(), result);
+    REQUIRE(result);
+    REQUIRE(result.getObject().isA<mx::RtPrim>());
+    mx::RtPrim bob(result.getObject().asA<mx::RtPrim>());
+    mx::RtRelationship rel1 = bob.createRelationship(mx::RtToken("rel1"));
+
+    mx::RtCommand::makeRelationship(rel1, add1, result);
+    REQUIRE(result);
+    mx::RtCommand::makeRelationship(rel1, add2, result);
+    REQUIRE(result);
+    REQUIRE(rel1.targetCount() == 2);
+
```

Comment:
Should you also test breaking relationships and failure condition for making/breaking ?
 Not sure if you can test the "command operands invalid" case as well ?